wavesrv/cmd/main-server.go

```diff
@@ -618,6 +619,15 @@ func HandleReadFile(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func HandleConfig(w http.ResponseWriter, r *http.Request) {
+	qvals := r.URL.Query()
+	filePath := qvals.Get("path")
```

Comment:
don't pull the filePath off a query value, pull it directly off the URL!  so i can request `/config/keybindings.json`